### PR TITLE
[KOGITO-7610] - Adding downstream content to upstream 

### PR DESCRIPTION
**Pull request by ricardozanini**
**JIRA:** https://issues.redhat.com/browse/KOGITO-7610

**Description:** In this JIRA we are adding the option to include content exclusive to downstream.

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+[[configurint-maven-rhbq]]
+== Configuring your Maven project to Red Hat build of Quarkus and OpenShift Serverless Logic
+
+To use the Red Hat Build of Quarkus (RHBQ) libraries you have to configure your local Maven installation.
+
+.Procedure
+. Follow the instructions in the RHBQ documentation, sections: link:{rhbq_config_maven_url}[Configuring the Maven settings.xml file for the online repository] and link:{rhbq_config_registry_url}[Configuring Quarkus extension registry client].
++
+. Add the following excerpt to your project's `pom.xml` file:
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc:14]: This is included before the Quarkus CLI/Maven plugin is executed, so either we split this in 2 parts where registry and Maven settings config will be included before the command and kogito-bom inclusion will be included after or here we can say to include this pom after the project is generated.

Comment (ricardozanini) [reply serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc:14]: I don't see it as a problem really. We may change the wording a bit, but splitting the procedure feels bad IMO. @hmanwani-rh what do you think?

File: serverlessworkflow/antora.yml [hunk 2]
```diff
@@ -6,6 +6,17 @@ nav:
...
+    # upstream: remove this var
+    #quarkus_cli_platform_redhat: -Pcom.redhat.quarkus.platform:quarkus-bom:2.7.6.Final-redhat-00006
+    # upstream: remove this var
+    #kogito_version_redhat: 1.24.0.Final-redhat-00001
+    # upstream: io.quarkus.platform
+    #quarkus_platform: com.redhat.quarkus.platform
+    quarkus_platform: io.quarkus.platform
+    # upstream: kogito-quarkus-serverless-workflow
+    # kogito_sw_gav: org.kie.kogito:kogito-quarkus-serverless-workflow:1.24.0.Final-redhat-00001
+    kogito_sw_gav: kogito-quarkus-serverless-workflow
```
Comment (jstastny-cz) [inline serverlessworkflow/antora.yml:18]: the value is not GAV now.

File: serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc (hunk 1 above)
Comment (jstastny-cz) [reply serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc:14]: I think something as `After project is created, add the following excerpt ...`

**Review by cristianonicolai — APPROVED** [review]

File: serverlessworkflow/antora.yml (hunk 2 above)
Comment (ricardozanini) [reply serverlessworkflow/antora.yml:18]: Upstream it's.

File: serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
+:page-partial:
+
+:rhbq_config_registry_url: https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/quarkus-2-7/guide/f93c45bd-4feb-4f74-a70a-022e9fb41957#_9064727c-4b8a-4068-a9ba-8de6b258a14a
+:rhbq_config_maven_url: https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/quarkus-2-7/guide/f93c45bd-4feb-4f74-a70a-022e9fb41957#proc_download-maven_quarkus-getting-started
+
+[[configurint-maven-rhbq]]
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc:6]:
```suggestion
[[proc-configuring-maven-rhbq]]
```

File: serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+== Configuring your Maven project to Red Hat build of Quarkus and OpenShift Serverless Logic
+
+To use the Red Hat Build of Quarkus (RHBQ) libraries you have to configure your local Maven installation.
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc:9]:
```suggestion
To use the Red Hat Build of Quarkus (RHBQ) libraries, you need to configure your local Maven installation.
```

File: serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+.Procedure
+. Follow the instructions in the RHBQ documentation, sections: link:{rhbq_config_maven_url}[Configuring the Maven settings.xml file for the online repository] and link:{rhbq_config_registry_url}[Configuring Quarkus extension registry client].
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc:12]:
```suggestion
. To configure the Maven `settings.xml` file and Quarkus extension registry client, follow the instructions in the link:{rhbq_config_maven_url}[Configuring the Quarkus Maven repository] and link:{rhbq_config_registry_url}[Configuring Quarkus extension registry client].
```

File: serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
++
+. After the project is created, add the following excerpt to your project's `pom.xml` file:
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc:14]:
```suggestion
. After the project is created, add the following dependencies to the `pom.xml` file of your project:
```

File: serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
++
+[source,xml,subs="attributes+"]
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc:16]:
```suggestion
.Dependencies in `pom.xml` file
[source,xml,subs="attributes+"]
```

File: serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc [hunk 8]
```diff
@@ -26,6 +26,10 @@ and https://marketplace.visualstudio.com/items?itemName=redhat.vscode-extension-
 
 For more information about the tooling and the required dependencies, see xref:getting-started/getting-familiar-with-our-tooling.adoc[tooling guide].
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc:27]:
```suggestion
For more information about the tooling and the required dependencies, see xref:getting-started/getting-familiar-with-our-tooling.adoc[Tooling in Serverless Workflow].
```

File: serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc [hunk 9]
```diff
@@ -26,6 +26,10 @@ and https://marketplace.visualstudio.com/items?itemName=redhat.vscode-extension-
...
 
+ifdef::quarkus_cli_platform_redhat[]
+include::../../pages/_common-content/productized-instructions.adoc[]
+endif::[]
+
 [[boostrapping-the-project]]
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc:33]:
```suggestion
[[proc-boostrapping-the-project]]
```

**Review by hmanwani-rh — COMMENTED** [review]
@ricardozanini I've added a few suggestions below. Please take a look and let me know if you've any questions.

**Comment by cristianonicolai** [comment]
@ricardozanini needs rebase

**Review by jstastny-cz — APPROVED** [review]
LGTM.

**Review by MarianMacik — APPROVED** [review]

**Review by akumar074 — APPROVED** [review]

**Pull request merged**